externals/service_ittnotify.h

```diff
@@ -96,10 +96,19 @@ class ScopedTask
     #define DAAL_ITTNOTIFY_SCOPED_TASK(name)                                            \
         static daal::internal::ittnotify::StringHandle __ittnotify_stringhandle(#name); \
         daal::internal::ittnotify::ScopedTask __ittnotify_task(__ittnotify_domain, __ittnotify_stringhandle)
+
 #else
+    #include "service_profiler.h"
+
+    #undef DAAL_ITTNOTIFY_CONCAT
+    #define DAAL_ITTNOTIFY_CONCAT(x, y)  DAAL_ITTNOTIFY_CONCAT2(x, y)
+    #define DAAL_ITTNOTIFY_CONCAT2(x, y) x##y
+
+    #define __DAAL_UNIQUE_ID __LINE__
+
     #define DAAL_ITTNOTIFY_DOMAIN(name)
-    #define DAAL_ITTNOTIFY_SCOPED_TASK(name)
+    #define DAAL_ITTNOTIFY_SCOPED_TASK(name) \
+        daal::internal::ProfilerTask DAAL_ITTNOTIFY_CONCAT(__, __DAAL_UNIQUE_ID) = daal::internal::Profiler::startTask(#name);
```

Comment:
```suggestion
        daal::internal::ProfilerTask DAAL_ITTNOTIFY_CONCAT(__profiler_taks__, __DAAL_UNIQUE_ID) = daal::internal::Profiler::startTask(#name);
```
It makes sense to add more descriptive name for the variable, will be helpful in case if debugging

Reply:
Done